app/store/configureStore.js

```diff
@@ -12,19 +12,17 @@ import createLogger from 'redux-logger';
  *                          rendering.
  */
 export default function configureStore(initialState, history) {
-  const middleware = [thunk, promiseMiddleware];
   // Installs hooks that always keep react-router and redux
   // store in sync
-  const reactRouterReduxMiddleware = routerMiddleware(history);
+  const middleware = [thunk, promiseMiddleware, routerMiddleware(history)];
   if (__DEVCLIENT__) {
-    middleware.push(reactRouterReduxMiddleware, createLogger());
-  } else {
-    middleware.push(reactRouterReduxMiddleware);
+    middleware.push(createLogger());
   }
 
-  const finalCreateStore = applyMiddleware(...middleware)(createStore);
-
-  const store = finalCreateStore(rootReducer, initialState);
+  const store = createStore(rootReducer, initialState, compose(
+    applyMiddleware(...middleware),
+    typeof window === 'object' && typeof window.devToolsExtension !== 'undefined' ? window.devToolsExtension() : f => f
```

Comment:
> It's only problematic if the end user has redux devtools installed, and even then only if you care about them seeing the internals of your JS, which shouldn't really have any secrets in it anyway.

Agreed.

Let's go with having it in production. If there are issues with it, I think we will find out really quickly.

#justdoit